### PR TITLE
BOAC-3106, /analytics page has user metrics per dept

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3106

* report on notes created in BOA 
* note count from various note-import jobs
* users report use b-table similar to Pax Manifest

### CODE REVIEW

File: src/components/reports/NotesReport.vue [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="pt-2">
+        <h4 class="font-size-16 font-weight-bold">Imported from Athletic Study Center</h4>
+        <div class="pl-3">
+          <div class="d-flex justify-content-between">
+            <div>
+              <label for="notes-count-sis">Total</label>
```
Comment (lyttam) [inline src/components/reports/NotesReport.vue]: @johncrossman  Should this be `for="notes-count-asc"`?

File: src/components/reports/NotesReport.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="pt-2">
+        <h4 class="font-size-16 font-weight-bold">Imported from Centers for Educational Equity and Excellence</h4>
+        <div class="pl-3">
+          <div class="d-flex justify-content-between">
+            <div>
+              <label for="notes-count-sis">Total</label>
```
Comment (lyttam) [inline src/components/reports/NotesReport.vue]: @johncrossman and here, `for="notes-count-ei"`?

Comment (johncrossman) [reply src/components/reports/NotesReport.vue]: Good catch @lyttam  Amending...

**Comment by johncrossman** [comment]
@lyttam - Ready for review

**Comment by lyttam** [comment]
Looks good, thanks!

**Pull request merged**